presto-parser/src/main/java/com/facebook/presto/sql/ExpressionFormatter.java

```diff
@@ -686,7 +687,7 @@ private static String formatIdentifier(String s)
     static String formatStringLiteral(String s)
     {
         s = s.replace("'", "''");
-        if (isAsciiPrintable(s)) {
+        if (CharMatcher.inRange((char) 0x20, (char) 0x7E).matchesAllOf(s)) {
```

Comment:
But now we have 2 definitions of "ascii printable". 
- here, inline
- `isAsciiPrintable(int codePoint)` method
..